README.md

```diff
@@ -173,6 +173,13 @@ cy.get('.select-payment-method')
   });
 ```
 
+#### Types multiple values in a sw-multi-select field and checks if the content was set correctly (multi select)
+
+```js
+cy.get('.select-payment-method')
+  .typeMultiSelectAndCheckMultiple(['Invoice', 'Paid in advance', 'Cash on delivery']);
```

Comment:
Would you like to create an issue assigned to me, dealing with updating the external shopware documentation? 🙂

Reply:
Only a ticket in NEXT :)